src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java

```diff
@@ -222,9 +223,14 @@ public JSONObject uploadRepo(String accessToken, JSONObject payload) {
             CodeSyncLogger.critical(String.format("Error while repo init, %s", error.getMessage()));
             return null;
         } catch (StatusCodeError statusCodeError) {
-            if (statusCodeError.getStatusCode()  == ErrorCodes.REPO_SIZE_LIMIT_REACHED) {
+
+            if (statusCodeError.getCustomErrorCode()  == CustomErrorCodes.PRIVATE_REPO_COUNT_LIMIT_REACHED) {
```

Comment:
nit, remove extra space before `==` @saadberry. 
You can run Format Code in the Intellij IDEA to fix such minor syntax issues

Reply:
Thanks. I will use that to check quality in the future